Module not found. I did ```npm i bkash-payment-gateway```. version: 2.0.0

Error: Cannot find module '...\node_modules\bkash-payment-gateway\dist\index.js'
    at createEsmNotFoundErr (node:internal/modules/cjs/loader:960:15)
    at finalizeEsmResolution (node:internal/modules/cjs/loader:953:15)
    at resolveExports (node:internal/modules/cjs/loader:482:14)
    at Function.Module._findPath (node:internal/modules/cjs/loader:522:31)
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:919:27)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)

--- a/.changeset/unlucky-apricots-fly.md
+++ b/.changeset/unlucky-apricots-fly.md
@@ -1,0 +1,5 @@
+---
+'bkash-payment-gateway': patch
+---
+
+fix: run build and tests before publish

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -15,7 +15,12 @@ jobs:
       - uses: actions/setup-node@v1
         with:
           node-version: 14
+      - run: sudo timedatectl set-timezone "Asia/Dhaka"
       - run: yarn ci
+      - run: yarn lint
+      - run: yarn formatcheck
+      - run: yarn build
+      - run: yarn test
 
       - name: Create Release Pull Request or Publish to npm
         uses: changesets/action@v1

